src/ui/components/tables/CCStressTable.vue

```diff
@@ -219,7 +218,7 @@ export default class CCStressTable extends Vue {
       case 2:
         return 2
       case 1:
-        return 3
+        return this.mech.CurrentStress <= 1 ? 4 : 3
```

Comment:
This is the change that fixes the listed issue. This has been brought in line with https://github.com/massif-press/compcon/blob/ecfbafba246f73ae1def076cd23ee677b3a2f076/src/ui/components/tables/CCStructureTable.vue#L279